Biz/Model/UserOTPForm.pm

```diff
@@ -50,6 +50,7 @@ sub internal_pre_execute {
 
 sub internal_validate_new {
     my($self) = @_;
+    # Not calling super as new password isn't a regular password
```

Comment:
It could be that we should remove OTP support. If we did "MFA", we'd use TOTP, which would be a different implementation. I'm not sure we'd every do that.